### PR TITLE
LPD-3481 - Table to show Tier Pricing after ther TIer is expired is always present

### PR DESCRIPTION
**Pull request by matyaswollner**
https://liferay.atlassian.net/browse/LPD-3481

### CODE REVIEW

**Comment by matyaswollner** [comment]
ci:test:sf

**Comment by matyaswollner** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-price-list-api/bnd.bnd
Comment (smottal) [inline modules/apps/commerce/commerce-price-list-api/bnd.bnd:1]: Please update the what commit description: Renamed method X to Y and A to B

**Review by smottal — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/commerce-price-list-api/bnd.bnd
Comment (matyaswollner) [reply modules/apps/commerce/commerce-price-list-api/bnd.bnd:1]: done

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:commerce-functional

**Comment by matyaswollner** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:sf

**Comment by matyaswollner** [comment]
ci:test:sf

**Comment by matyaswollner** [comment]
ci:test:relevant

**Comment by matyaswollner** [comment]
ci:test:commerce-functional

**Comment by smottal** [comment]
ci:forward

**Pull request closed without merging**